app/src/main/kotlin/com/wire/android/ui/common/Extensions.kt

```diff
@@ -49,6 +52,12 @@ internal fun dimensions() = MaterialTheme.wireDimensions
 @Composable
 internal fun colorsScheme() = MaterialTheme.wireColorScheme
 
+@Composable
+internal fun WireColorScheme.conversationColor(id: ConversationId): Color {
+    val colors = colorsScheme().groupAvatarColors
```

Comment:
I think either you take the current `WireColorScheme`, the one for which this function is called:
```
fun WireColorScheme.conversationColor(id: ConversationId): Color {
    val colors = this.groupAvatarColors
    ....
```
or don't make it an extension function:
```
fun conversationColor(id: ConversationId): Color {
    val colors = colorsScheme().groupAvatarColors
    ...
```
😄